Details for supported AAD prov scenarios

AlexFilipin opened the pull request:
Added information is the result of Microsoft Support Case REG:118030917783604 and has been confirmed with the back-end team / product group from v-yutiru@microsoft.com. Took us >2 weeks of support to figure this out because in theory it would be possible to run provisioning of cloud only accounts side by side with DirSync

azurecla commented:
Hi __@rcarmo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rucarmo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@fsautomata - Elio, please review this PR from @rcarmo in the public repo and let me know whether I should merge it.

AlexFilipin committed:
Details for supported AAD prov scenarios

Added information is the result of Microsoft Support Case REG:118030917783604 and has been confirmed with the back-end team / product group from v-yutiru@microsoft.com. Took us >2 weeks of support to figure this out because in theory it would be possible to run provisioning of cloud only accounts side by side with DirSync
